Week-2/Html Basics/htmlTables.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HTML Tables</title>
+</head>
+<body>
+    <table border="1">
+        <tr>
+            <td colspan="3"><h1>1</h1></td>
+        </tr>
+        <tr>
+            <td colspan="1" rowspan="2"><h1>2</h1></td>
+            <td><h1>3</h1></td>
+            <td><h1>4</h1></td>
+        </tr>
+        <tr>
+            <td><h1>5</h1></td>
+            <td><h1>6</h1></td>
```

Comment:
Tables are meant for tabular data, like spreadsheets, and should be structured accordingly. If you want to create a layout or section headings, it's better to use other HTML elements such as `<div>` or semantic HTML elements like `<section> and <article>` for content organization.


```
<div>
    <h1>1</h1>
</div>

<table border="1">
    <tr>
        <td><h1>2</h1></td>
        <td><h1>3</h1></td>
        <td><h1>4</h1></td>
    </tr>
    <tr>
        <td><h1>5</h1></td>
        <td><h1>6</h1></td>
    </tr>
</table>
```